apps/src/applab/levels.js

```diff
@@ -87,6 +89,10 @@ levels.ec_simple = {
       'color': 'yellow',
       'blocks': []
     },
+    'Storage': {
+      'color': 'orange',
```

Comment:
yes, orange just worked because I reused an existing color. I'm assuming these will all be getting new colors at some point.